src/Test/JWKSHelper.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopware\App\SDK\Test;
+
+use Lcobucci\JWT\Signer\CannotSignPayload;
+use Lcobucci\JWT\Signer\Key\InMemory;
+use Lcobucci\JWT\Signer\Rsa;
+use Lcobucci\JWT\Signer\Rsa\Sha256;
+use Lcobucci\JWT\Signer\Rsa\Sha384;
+use Lcobucci\JWT\Signer\Rsa\Sha512;
+use Lcobucci\JWT\Token\DataSet;
+use Lcobucci\JWT\Token\Plain;
+use Lcobucci\JWT\Token\Signature;
+
+/**
+ * Helper for testcases during which creating a valid signed JWT is required
+ *
+ * @codeCoverageIgnore
+ * @internal
+ */
+final class JWKSHelper
+{
+    /**
+     * @param array<array-key, mixed> $headers
+     * @param array<array-key, mixed> $claims
+     */
+    public static function encodeIntoToken(array $headers, array $claims, string $signer = 'RS256'): Plain
```

Comment:
Why should we provide such a function?

Reply:
This is a handy helper to create a working encoded token during tests with a fixed fixture JWKS and private key